changelog.d/11943.doc

```diff
@@ -0,0 +1 @@
+Recommend upgrading treq alongside twisted.
```

Comment:
advise *against* a changelog file for a doc change targetting `master`: it won't make any sense to include this for 1.53.0.